test/run

```diff
@@ -811,6 +918,10 @@ compileTest() {
   for f in ${compile_dir}/.profile.d/*; do source $f > /dev/null 2> /dev/null ; done
 
   capture ${bp_dir}/bin/test ${compile_dir}
+
```

Comment:
Fixes a bug in the `compileTest` function that would cause subsequent tests to fail